docker/Makefile

```diff
@@ -66,11 +60,11 @@ pdo-dev-image:
 	@echo "Building pdo-dev docker image"
 	docker build $(DOCKER_BUILD_OPTS) -f Dockerfile.pdo-dev -t pdo-dev .
 
-pdo-composition: pdo-dev-image
-	@echo "Running PDO docker composition for Sawtooth"
-	env PDO_REPO_BRANCH=$$(git rev-parse --abbrev-ref HEAD) $(DOCKER_COMPOSE_COMMAND) $(DOCKER_COMPOSE_OPTS_STL) build
+pdo-composition-ccf: pdo-dev-image
+	@echo "Running PDO docker composition for CCF"
+	env PDO_REPO_BRANCH=$$(git rev-parse --abbrev-ref HEAD) $(DOCKER_COMPOSE_COMMAND) $(DOCKER_COMPOSE_OPTS_CCF) build
 	# Note:
-	# - using `sawtooth-pdo.local-code.yaml` in above will cause the docker context to be `../`.
+	# - using `ccf-pdo.local-code.yaml` in above will cause the docker context to be `../`.
 	#   To make sure that we do not pull in (too much) unnecessary stuff, we explicitly excludes
```

Comment:
Same.

Reply:
Here I would not say the same.
The yaml contains specific stuff about ccf and pdo. We need some more refactoring to separate the two.

Reply:
while i basically agree. this is not the PR for it.